docs/source/index.rst

```diff
@@ -65,23 +66,23 @@ outlines recent changes, new features, and future development plans.
 Getting involved
 ================
 
-Cartopy was originally developed at the UK Met Office to allow scientists to visualise 
-their data on maps quickly, easily and most importantly, accurately. 
+Cartopy was originally developed at the UK Met Office to allow scientists to visualise
```

Comment:
I believe all of these are just trailing whitespace updates that my editor apparently auto-corrected. LMK if you want me to revert these for some reason